docs/docs/policies.mdx

```diff
@@ -320,6 +338,15 @@ However, additional parameters exist that can be adapted.
 | entity_recognition                    | True                   | If 'True' entity recognition is trained and entities are     |
 |                                       |                        | extracted.                                                   |
 +---------------------------------------+------------------------+--------------------------------------------------------------+
+| split_entities_by_comma               | True                   | Splits a list of extracted entities by comma to treat each   |
+|                                       |                        | one of them as a single entity. Can either be `True`/`False` |
+|                                       |                        | globally, or set per entity type, such as:                   |
+|                                       |                        | ```                                                          |
+|                                       |                        | - name: TEDPolicy                                            |
+|                                       |                        |   split_entities_by_comma:                                   |
+|                                       |                        |     address: True                                            |
+|                                       |                        | ```                                                          |
++---------------------------------------+------------------------+--------------------------------------------------------------+
```

Comment:
I don't think we need the docs in the hidden table, given its already described in the "visible" docs (Neither of the other options in the above list are listed in the table below).

Reply:
Ignore me, I have just seen @Ghostvv's comment above and have checked the table again